c2corg_ui/static/js/map/map.js

```diff
@@ -59,7 +59,14 @@ app.module.directive('appMap', app.mapDirective);
  * @export
  * @ngInject
  */
-app.MapController = function($scope, mapFeatureCollection) {
+app.MapController = function($scope, mapFeatureCollection, $attrs) {
+
+
+  /**
+   * @type {number}
+   * @private
+   */
+  this.zoom_ = $attrs['zoom'];
```

Comment:
Well, OK for this change but please keep the consistency of the map directive attributes (see my comment above about the scope param).